kuzzleio/kuzzle
# Use document mGet route in GET

## What does this PR do ?


This PR permits to use the mGet route in GET.
Problem was: the route is only usable in POST, which responses are not possible to cache with Service worker Cache Storage API
https://github.com/kuzzleio/kuzzle/issues/1366

ticket [Jira](https://jira.kaliop.net/browse/KZL-1336)


### How should this be manually tested?

As its a GET route, Ids are meant to be passed in query string separated by a comma.

  - Step 1 : Create documents with id `foo` and `bar`.
  - Step 2 : Get them by hitting `http://localhost:7512/nyc-open-data/yellow-taxi/_mGet?ids=foo,bar` with GET verb.
  - Step 3 : You can also verify that the POST way to hit the route still works.
### Other changes



### Boyscout
